src/middlewares/validateAuth.js

```diff
@@ -3,26 +3,31 @@ import joiValidator from '../helpers/joiValidator';
 import { respondWithWarning } from '../helpers/responseHandler';
 
 /**
+ *   Class to validate user login and signup forms submission
+ */
+class ValidateAuth {
+  /**
    * validate email and password
    * @param {Object} req
    * @param {Object} res
    * @param {Function} next
    * @returns {Object} error
    */
-const validateSigninFormData = (req, res, next) => {
-  const password = Joi.string().required();
-  const email = Joi.string().email().required();
+  static validateSigninFormData(req, res, next) {
```

Comment:
@SteveFunso Kindly revert these changes. using classes is not recommended in this case.